controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -1195,6 +1198,114 @@ func createMultiDcClusterWithStargate(t *testing.T, ctx context.Context, f *fram
 	verifyObjectDoesNotExist(ctx, t, f, sg2Key, &stargateapi.Stargate{})
 }
 
+// changeNumTokensValue creates a Datacenter and then changes the numTokens value
+// Such change is prohibited and should fail
+func changeNumTokensValue(t *testing.T, ctx context.Context, f *framework.Framework, namespace string) {
+	require := require.New(t)
+
+	k8sCtx := "cluster-1"
```

Comment:
The integration test framework creates 3 clusters - cluster-0, cluster-1, cluster-2. cluster-0 is implicitly the control plane cluster.